config/initializers/session_store.rb

```diff
@@ -1,3 +1,7 @@
 # Be sure to restart your server when you modify this file.
 
-Rails.application.config.session_store :cookie_store, key: '_event_x_session'
+Rails.application.config.session_store :cookie_store, key: '_event_x_session',
+domain: {
+  production: "event-ex.herokuapp.com",
+  development: ".lvh.me"
+  }.fetch(Rails.env.to_sym, :all)
```

Comment:
Indent the right brace the same as the start of the line where the left brace is.